src/lib/FreeC/Backend/Agda/Converter/Free.hs

```diff
@@ -6,9 +6,12 @@ module FreeC.Backend.Agda.Converter.Free
   , free
   , applyFreeArgs
   , addFreeArgs
+  , freeDataDecl
```

Comment:
Since `freeDataDecl` is not of general purpose, it should be part of `FreeC.Backend.Agda.Converter.TypeDecl`.